guide_drafts/backend_installation.md

```diff
@@ -19,13 +19,12 @@ For example, to install with sqlite only, run:
 cargo install diesel_cli --no-default-features --features sqlite
 ```
 
-For projects that depend on diesel,
-you can specify which backends are required in the `Cargo.toml`.
+To install Diesel CLI with support for more than one backend,
+use spaces to separate the features.
```

Comment:
My spidey senses are tingling. There should be a link here. To the cargo-install docs. But… where are they? [Here](https://doc.rust-lang.org/nightly/cargo/)? [H̸ere](https://github.com/rust-lang/cargo/blob/19cfb6742f5ff7a53220056dccdfd5626b647428/src/etc/man/cargo-install.1)?? [Here̴](https://github.com/rust-lang/cargo/blob/75f8541d32b276ef0764535fcfb63befbc285126/src/bin/install.rs#L40-L103)??? Cargo team, _where_ a̢re the `cargo ins̸ţall` docs?̵?̶!́?